tests/e2e/specs/http.spec.ts

```diff
@@ -31,7 +31,8 @@ test.describe("requests", () => {
 
     test("returns wrapped proxy route when hit", async ({ request }) => {
       const req = await request.get("./custom");
-      expect(req.headers()["x-test-pageid"]).toBe("/proxy/pages");
+      const req = await request.get("./json-route");
+      expect(req.headers()["x-test-pageid"]).toBe("/proxy/pages/wrapped");
```

Comment:
Nit: Do we expect this to be `/proxy/pages/passthru` in another case? If so, can we test that as well?